modules/serverless-eventing-components.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * serverless-knative-eventing.adoc
+
+[id="serverless-eventing-components_{context}"]
+= Knative Eventing components
+
+A typical Knative Serving deployment on {ServerlessProductName} follows this basic workflow.
```

Comment:
Not Knative Eventing?

Reply:
Yes, it should be. Derp. Thank you :slightly_smiling_face: